### PR TITLE
Feature/003 Create&edit interface -> dev

### PR DESCRIPTION
**Pull request by WMinowski**

### CODE REVIEW

File: MVC-App/Controllers/tblRelationsController.cs [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using MVC_App.Domain.Models;
+
+namespace MVC_App.Controllers
+{
+    public class tblRelationsController : Controller
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: rename
tblRelationsController -> RelationsController 
tblRelationsController.cs -> RelationsController.cs

File: MVC-App/Controllers/tblRelationsController.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    {
+        private testEntities db = new testEntities();
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: private testEntities db;

File: MVC-App/Controllers/tblRelationsController.cs [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+        private IEnumerable<RelationModel> relationModels;
+
+        public tblRelationsController()
+        {
+            var relations = from tblRelation in db.tblRelation
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: simplify
relationModels =  from tblRelation in db.tblRelation ......... .ToList();

File: MVC-App/Controllers/tblRelationsController.cs [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+                            join tblRelationAddress in db.tblRelationAddress on tblRelation.Id equals tblRelationAddress.RelationId
+                            where tblRelation.IsDisabled != true
+                            select new RelationModel
+                            {
+                                Id = tblRelation.Id,
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: tblRelation -> Relation

File: MVC-App/Controllers/tblRelationsController.cs [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+                                RelationAddressId = tblRelationAddress.Id,
+                                Name = tblRelation.Name,
+                                FullName = tblRelation.FullName,
+                                TelephoneNumber = tblRelation.TelephoneNumber,
+                                Email = tblRelation.EMailAddress,
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: tblRelationAddress-> RelationAddress

File: MVC-App/Controllers/tblRelationsController.cs [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+        }
+
+        // GET: tblRelations/Details/5
+        public async Task<ActionResult> Details(Guid? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+            tblRelation tblRelation = await db.tblRelation.FindAsync(id);
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: use var where it is possible
var tblRelation = await db.tblRelation.FindAsync(id);

File: MVC-App/Controllers/tblRelationsController.cs [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+                    EMailAddress = relationModel.Email,
+                    CreatedAt = DateTime.Now,
+                    CreatedBy = "admin",
+                    IsDisabled = false,
+                    IsTemporary = false,
+                    IsMe = false,
+                    PaymentViaAutomaticDebit = false,
+                    InvoiceDateGenerationOptions = 0,
+                    InvoiceGroupByOptions = 0
+                });
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/tblRelationsController.cs]: refactor: 
var relation = new tblRelation {
                    Id = relationModel.Id,
                    Name = relationModel.Name,
                    FullName = relationModel.FullName,
                    TelephoneNumber = relationModel.TelephoneNumber,
                    EMailAddress = relationModel.Email,
                    CreatedAt = DateTime.Now,
                    CreatedBy = "admin",
                    IsDisabled = false,
                    IsTemporary = false,
                    IsMe = false,
                    PaymentViaAutomaticDebit = false,
                    InvoiceDateGenerationOptions = 0,
                    InvoiceGroupByOptions = 0
                };

db.tblRelation.Add(relation);

**Review by dmyhun-andersen — APPROVED** [review]

File: MVC-App/Controllers/tblRelationsController.cs (hunk 2 above)
Comment (WMinowski) [reply MVC-App/Controllers/tblRelationsController.cs]: non-initialised variable generates exception at db.Dispose(), so I left as it is

**Pull request merged**